ast/ast.go

```diff
@@ -747,7 +747,11 @@ func (i *PipeLiteral) Copy() Node {
 // StringLiteral expressions begin and end with double quote marks.
 type StringLiteral struct {
 	BaseNode
+	// Value is the unescaped value of the string literal
 	Value string `json:"value"`
+	// Literal may be empty.
+	// When not empty it represents the literal string found in the source including quotes.
+	Literal string `json:"literal,omitempty"`
```

Comment:
This should already be contained in the `BaseNode` within the `Source` attribute.